NineChronicles.Headless.Executable/Program.cs

```diff
@@ -357,8 +357,22 @@ static void ConfigureSentryOptions(SentryOptions o)
 
                 IActionTypeLoader MakeStaticActionTypeLoader() => new StaticActionTypeLoader(
                     Assembly.GetEntryAssembly() is { } entryAssembly
+#if LIB9C_DEV_EXTENSIONS
+                        ? new[]
+                        {
+                            typeof(ActionBase).Assembly,
+                            typeof(Lib9c.DevExtensions.Action.CreateOrReplaceAvatar).Assembly,
```

Comment:
If we add another DevEx actions, do we have to add all individual actions like this?

Reply:
No, it adds the `Lib9c.DevExtensions`' assembly, not only an action. If you add another DevEx action to `Lib9c.DevExtensions` project, it will be added automatically. If you create a new project, it should be added like this.